config/routes.rb

```diff
@@ -4,12 +4,13 @@
   get 'unauthorised' => 'pages#unauthorised'
   #Users routes
   resources :users do
-    resources :accounts
+    resources :transactions
+    resources :accounts 
   end
-
   #Session routes
   get '/login', to: 'sessions#login'
```

Comment:
I don't see a `def login` in your sessions controller so i think this line can be removed.

Reply:
I think its all to do with a gem I'm using, when I delete either login route my code fails